plugins/org.jboss.reddeer.gef/src/org/jboss/reddeer/gef/matcher/IsEditPartWithParent.java

```diff
@@ -0,0 +1,57 @@
+package org.jboss.reddeer.gef.matcher;
+
+import org.eclipse.gef.EditPart;
+import org.hamcrest.BaseMatcher;
+import org.hamcrest.Description;
+
+/**
+ * Returns true if an edit part is a child of a given parent.
+ * 
+ * @author Andrej Podhradsky (apodhrad@redhat.com)
+ *
+ */
+public class IsEditPartWithParent extends BaseMatcher<EditPart> {
```

Comment:
shouldnt this be called EditPartWithParent (without "Is") ? Seems to be more aligned with names of other matches